tests/integration/test_integration_snapshots.py

```diff
@@ -292,3 +293,32 @@ def test_tracetagsprocessor_only_adds_new_tags():
         span.set_metric(SAMPLING_PRIORITY_KEY, USER_KEEP)
 
     tracer.shutdown()
+
+
+# Override the token so that both parameterizations of the test use the same snapshot
+# (The snapshots should be equivalent)
+@snapshot(wait_for_num_traces=1, token_override="tests.integration.test_integration_snapshots.test_env_vars")
+@pytest.mark.parametrize("use_ddtracerun", ["yes", "no"])
+def test_env_vars(use_ddtracerun, ddtrace_run_python_code_in_subprocess, run_python_code_in_subprocess):
```

Comment:
This test fails in python3.5: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/25487/workflows/4a0464ee-09d5-498d-ac15-717a85a68c36/jobs/1780306